echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/PipelineCache.java

```diff
@@ -66,45 +76,68 @@ public class PipelineCache implements MonitoredPoller {
   public PipelineCache(
       @Value("${front50.polling-interval-ms:30000}") int pollingIntervalMs,
       @Value("${front50.polling-sleep-ms:100}") int pollingSleepMs,
-      @Value(
-              "${pipelineCache.parallelism:#{T(java.lang.Runtime).getRuntime().availableProcessors()}}")
-          int planParallelism,
+      @NonNull PipelineCacheConfigurationProperties pipelineCacheConfigurationProperties,
       ObjectMapper objectMapper,
       @NonNull Front50Service front50,
       @NonNull OrcaService orca,
-      @NonNull Registry registry) {
+      @NonNull Registry registry,
+      @NonNull List<BaseTriggerEventHandler> triggerHandlers) {
```

Comment:
Base vs. plain TriggerEventHandler?

Reply:
Using TriggerEventHandler here would give me ManualEventHandler, where BaseTriggerEventHandler doesn't.  Since pipelines don't typically define triggers whose type is `"manual"` it made sense to leave them out of here.

Reply:
Yeah only worry is a plugin or someone else who extends non-base.  Wondering if we could add an explicit exlucsion for manual trigger types or something.  I'm not as sure on this one.... I can leave it base, just one of those things that caught my eye.

Reply:
Yeah, I'm not 100% how this will play out.  With any luck folks who enable the feature flag and have trouble will get in touch.